.gitignore

```diff
@@ -3,3 +3,5 @@
 tmp/
 output/
 node_modules/
+
+editions/tw5.com/tiddlers/$__StoryList.tid
```

Comment:
hmmm, not sure about this one. ... There are 6 "-server" editions and every one of them will create a $__StoryList.tid ... So we may create a separated PR to fix this behaviour. ...

I think we should be able to safely ignore every tiddler `$__StoryList.tid` for core editions. 

There is a problem if the DefaultTiddler is set to `[list[$:/StoryList]]` which prevented us from disabling the story list save at the TW saver level. ... But for core editions, DefaultTiddler isn't set.